locator/network_topology_strategy.cc

```diff
@@ -307,9 +307,12 @@ effective_replication_map_ptr network_topology_strategy::make_replication_map(ta
 //    initial_tablets = max(nr_shards_in(dc) / RF_in(dc) for dc in datacenters)
 //
 
-static unsigned calculate_initial_tablets_from_topology(const schema& s, token_metadata_ptr tm, const std::unordered_map<sstring, size_t>& rf) {
+static unsigned calculate_initial_tablets_from_topology(const schema& s, token_metadata_ptr tm,
+                                                        const std::unordered_map<sstring, size_t>& rf,
+                                                        unsigned initial_scale) {
```

Comment:
note: it makes sense to support 0 < initial_scale < 1 too. initial_scale = 0.3 -> this table needs to be on a third of all shards